app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -11,16 +11,25 @@ package com.github.se.cyrcle.model.user
  * @property profilePictureUrl URL of the profile picture of the user.
  * @property favoriteParkings List of unique identifiers of the user's favorite parkings.
  */
-data class User(
+data class UserPublic(
```

Comment:
Don't forget to update documentation here

Reply:
Will do, and add more documentation for your other comment about doc.